P5/Source/Guidelines/en/BIB-Bibliography.xml

```diff
@@ -70,6 +76,10 @@ $Id$
       <bibl xml:id="NZETC01">
         <author>Baker, James K.</author>. <title>Night in Tarras</title>. In <title level="j">Hilltop: A Literary
       Paper</title>, vol 1 no 2. Wellington: Victoria University College Literary Society. (<date>1949</date>). </bibl>
+      <bibl xml:id="COEDADD-eg-93"><title>Base de datos paleográfica da lírica galego-portuguesa (PalMed)</title>.
+        Versión 1.2. <pubPlace>Santiago de Compostela</pubPlace>: <publisher>Centro Ramón Piñeiro para
+          a Investigación en Humanidades</publisher>.
```

Comment:
```suggestion
          a Investigación en Humanidades</publisher>. <ref target="http://www.cirp.gal/palmed">http://www.cirp.gal/palmed</ref>. 
```
It doesn’t seem possible to provide the URL of the `<citedRange>` (I guess that it is due to how the queries are processed) but at least we can add the URL to the database.